### DISCUSSION TITLE
concepts/auth/

### DISCUSSION THREAD

**Discussion by giscus** [go-gost/gost#596] [General]
# concepts/auth/

https://gost.run/concepts/auth/

**Comment by flexmark** [comment]
{"handler":"auto","kind":"service","level":"error","listener":"tcp","msg":"authenication failed","service":"service-0","sid":"cs6hcer5t88pk91djq70","time":"2024-10-14T21:01:47.217+08:00"}

authenication lost t ,  authentication!

**Comment by flexmark** [comment]
Yes，but not for system sshd



发自我的iPhone


------------------ Original ------------------
From: seinaville ***@***.***&gt;
Date: Sat,Oct 26,2024 11:37 PM
To: go-gost/gost ***@***.***&gt;
Cc: flexmark ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [go-gost/gost] concepts/auth/ (Discussion #596)





 
Can we use the module "authers" to verifier ssh or sshd? Thank you.

**Comment by permheaddamage** [comment]
外部文件作为数据源的时候，可以用base64密文吗？

**Comment by m7rick** [comment]
{"caller":"service/service.go:181","handler":"http","kind":"service","level":"error","listener":"tcp","msg":"accept: accept tcp [::]:19001: use of closed network connection","service":"proxy-service","time":"2024-12-06T23:31:29.686+08:00"}
